src/js/easymde.js

```diff
@@ -1945,7 +1945,12 @@ EasyMDE.prototype.render = function (el) {
         this.gui.statusbar = this.createStatusbar();
     }
     if (options.autosave != undefined && options.autosave.enabled === true) {
-        this.autosave();
+        this.autosave(); // use to load localstorage content
+        this.codemirror.on('change', function () {
+            setTimeout(function () {
```

Comment:
I don't think this works the way you expect it to.
Now on every 'change' (keystroke) an autosave will take place after 1 second. This means that if I type multiple characters, it will save the contents of the editor multiple times, instead of saving once every second.

Maybe we should be looking at something like a 'debounce' system, where it only saves one time if any amount of changes have happened within the set delay.

Reply:
```js
clearTimeout(this._autosave_timeout)
this._autosave_timeout = setTimeout(function () {
```

clearTimeout should be fine